diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleWarnTest.kt

```diff
@@ -606,5 +606,29 @@ class IndentationRuleWarnTest : LintTestBase(::IndentationRule) {
         )
     }
 
+    @Test
+    @Tag(WarningNames.WRONG_INDENTATION)
+    fun `should trigger on string templates starting with new line`() {
+        lintMethod(
+            """
+                |fun foo(some: String) {
+                |    fun bar() {
+                |        val a = "${'$'}{
+                |        expression
```

Comment:
I corrected `expression` as it should be, but then it incorrectly detects the indentation of the bottom lines

Reply:
Fixed